apps/demos/Demos/DataGrid/Column3RdPartyEngineTemplate/jQuery/index.html

```diff
@@ -18,7 +18,7 @@
       <div id="gridContainer"></div>
       <script id="gridPhoto" type="text/html">
         <div class="photo">
-          <img src="{{ value }}" />
+          <img src="{{ value }}" alt="Image of {{ data.FirstName }} {{ data.LastName }}" />
```

Comment:
```suggestion
          <img src="{{ value }}" alt="Picture of {{ data.FirstName }} {{ data.LastName }}" />
```

There's not much difference in "picture" and "image" but in this context picture suits better (since it's a photo and we're not talking about technical aspects).